app/script/controllers/product.controller.js

```diff
@@ -1,18 +1,12 @@
 (function () {
     angular.module('app')
-        .controller('productController', function ($http, $scope) {
-            $http({
-                url:'http://localhost:1337/product',
-                method:'GET'
-
-            })
+        .controller('productController', function ($scope, apiService) {
+            apiService.getProducts()
                 .then(function(response){
-                    $scope.product=response.product;
+                    $scope.product = response.data;
```

Comment:
please rename it to products, since it is an array of products. (you will need to rename it in the template as well)